### ISSUE TITLE
Executable `prettier` not found on Windows

### ISSUE THREAD

**Issue by EnchoMishinevQC** [Quantco/pre-commit-mirrors-prettier#11]
Installing and attempting to run the hook on a fresh repository on a Windows machine results in:
```
prettier-conda...........................................................Failed
- hook id: prettier-conda
- exit code: 1

Executable `prettier` not found
```

**Comment by JanJaguschQC** [comment]
what `prettier` version are you using, @EnchoMishinevQC?

this should have been fixed since https://github.com/conda-forge/prettier-feedstock/pull/31

**Comment by EnchoMishinevQC** [comment]
Using `2.6.2`, is the fix only in `2.7.1`?

**Comment by JanJaguschQC** [comment]
> Using `2.6.2`, is the fix only in `2.7.1`?

yes, upgrading should resolve the issue

**Comment by ivergara** [comment]
> Using `2.6.2`, is the fix only in `2.7.1`?

The "fix" only applies to the latest version available at the time.

**Comment by EnchoMishinevQC** [comment]
Fixed in latest version ✅
